tests/console/docker.pm

```diff
@@ -10,13 +10,17 @@
 
 # Summary: Test docker installation and extended usage
 #    Cover the following aspects of docker:
-#      * package can be installed
-#      * daemon can be started
+#      * docker package can be installed
+#      * docker daemon can be started
 #      * images can be searched on the Docker Hub
 #      * images can be pulled from the Docker Hub
-#      * containers can be spawned, started on background, stopped, deleted
-#      * images can be deleted
+#      * local images can be listed
+#      * containers can be spawned
+#      * containers can be run on background
+#      * containers can be stopped
 #      * network is working inside of the containers
+#      * containers can be deleted
+#      * images can be deleted
 # Maintainer: Petr Cervinka <pcervinka@suse.com>, Flavio Castelli <fcastelli@suse.com>
```

Comment:
I don't feel as maintainer anymore :)

Reply:
I have the same question. As I said before, I haven't added nor removed any instruction. I have only reorganized the existing ones. I have only edited the comments.

My plan after this PR get merged is to fix some issues like false negatines, the one @okurz mentioned, the one you mentioned and so on...

Reply:
I will remove you then in the next PR and add myself ;)

Reply:
I can remove you in the next PR where I will a lot of issues like the one you mentioned and add myself as maintainer

Reply:
@SergioAtSUSE  feel free to add me as maintainer.